src/view/component/recommendedPackage/recommendedPackage.jsx

```diff
@@ -19,6 +19,16 @@ const defaultProps = {
   isLoading: false,
 };
 
+export const hasToScrollToDisneyHotel = (currentHotelId = '', data = {}) => {
+  const { disneyHotels = [] } = data;
+  return currentHotelId === '' && !!disneyHotels.length;
+};
+
+export const hasToScrollToPartnerHotel = (data = {}) => {
+  const { disneyHotels = [], partnerHotels = [] } = data;
+  return !disneyHotels.length && !!partnerHotels.length;
```

Comment:
Disclaimer : Ceci est un avis subjectif.
Je trouve cette function peu lisible.

Reply:
Si je fais : 
```suggestion
  return !hasDisneyHotels() && !hasPartnerHotels();
```
ça te convient ?

Reply:
oui ça me parle :)

Reply:
isNotEmpty (?)